lib/timeval.c

```diff
@@ -84,6 +84,37 @@ struct curltime curlx_tvnow(void)
   return cnow;
 }
 
+#elif defined(__APPLE__)
+
+#include <stdint.h>
+#include <mach/mach_time.h>
+
+struct curltime curlx_tvnow(void)
```

Comment:
Also beware that I sneakily just renamed that function to `Curl_now()` since 5d543fe906 =)